src/components/Form/FormFields/TextFormField.tsx

```diff
@@ -32,6 +32,8 @@ export type TextFormFieldProps = FormFieldBaseProps<string> &
     trailingPadding?: string | undefined;
     leadingPadding?: string | undefined;
     suggestions?: string[];
+    min?: number | string;
+    max?: number | string;
```

Comment:
this is given by `DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>` right? Do we need to redefine it here?